docs/articles/documentation/using-testcafe/using-testcafe-docker-image.md

```diff
@@ -45,13 +60,17 @@ This command takes the following parameters:
 * `-it testcafe/testcafe` - runs TestCafe in the interactive mode with the console enabled;
 * `${TESTCAFE_ARGS}` - arguments passed to the `testcafe` command. You can use any arguments from the TestCafe [command line interface](command-line-interface.md);
 
-    `-it testcafe/testcafe 'chromium --no-sandbox,firefox' /tests/test.js`
+    ```sh
+    -it testcafe/testcafe 'chromium --no-sandbox,firefox' /tests/test.js
+    ```
 
     You can run tests in the Chromium and Firefox browsers preinstalled to the Docker image. Add the `--no-sandbox` flag to Chromium if the container is run in the unprivileged mode.
```

Comment:
Change "in the Chromium" to "in Chromium"
Change "preinstalled" to "pre-installed"
By "to the Docker image" do you mean "in the Docker image"